packages/clerk-js/src/ui/localization/applyTokensToString.ts

```diff
@@ -66,15 +95,36 @@ const parseTokensFromLocalizedString = (
     // replace it with its localized value in the next step
     normalisedString = normalisedString.replace(/{{.+?}}/, `_++${token}++_`);
   });
-  return { expressions, normalisedString };
+  return { expressions: expressions as TokenExpression[], normalisedString };
 };
 
 const applyTokenExpressions = (s: string, expressions: TokenExpression[], tokens: Tokens) => {
   expressions.forEach(({ token, modifiers }) => {
-    const value = modifiers.reduce((acc, mod) => MODIFIERS[mod](acc), tokens[token]);
+    const value = modifiers.reduce((acc, mod) => {
+      try {
+        return MODIFIERS[mod.modifierName](acc, ...mod.params);
+      } catch (e) {
+        console.warn(e);
+        return '';
+      }
+    }, tokens[token]);
     s = s.replace(`_++${token}++_`, value);
   });
   return s;
 };
 
 const assertKnownModifier = (s: any): s is Modifier => Object.prototype.hasOwnProperty.call(MODIFIERS, s);
+
+const getModifierWithParams = (modifierExpression: string): { modifierName: string; params: string[] } => {
```

Comment:
✅

Reply:
We usually prefer to let TS infer the returned type instead of explicitly specifying it  for internal methdos
```suggestion
const getModifierWithParams = (modifierExpression: string) => {
```